great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -193,8 +177,8 @@ def name(self) -> str:
         return self._name
 
     @property
-    def data_context(self) -> BaseDataContext:
-        return self._data_context
+    def validator(self) -> Validator:
```

Comment:
is this needed, or can we just keep it private?

Reply:
@donaldheppner It is not needed -- we can keep it private (if works, will make the change) -- thanks!